neuralmonkey/dataset.py

```diff
@@ -171,6 +171,8 @@ def load_dataset_from_files(
     Keyword arguments:
         name: The name of the dataset to use. If None (default), the name will
               be inferred from the file names.
+        primary_series: A series that is used for measuring (token_level)
+                        batch size, bucketing, etc.
```

Comment:
tož čo je toto?

Reply:
to tam vubec nema byt

Reply:
To by tam ale mělo být ne. To je potřeba nějak říct, podle čeho chci bucketovat, ne?

Reply:
Spis dava smysl tam mit ignore prepinac, kterym reknes, ktere series nema pri bucketovani zohlednovat.

Reply:
Takhle si ale muzes lehce odhadnout, kolik examplu muzes do batche narvat, protoze vis, ze v te batchi nebude > <pocet_series> * <velikost_examplu/bucketu> * <pocet_examplu>
Kde velikost_examplu se vzdy bere podle nejvetsiho examplu v batchi.

Je to obecne funkcni reseni. Pokud chces mit nejak chytreji ohandlovane obrazky, tak ti nic nebrani v tom, si to nalezite rozsirit.